libs/cluster/Server/ClusterManagerSlotState.cs

```diff
@@ -419,14 +419,12 @@ public bool TryPrepareSlotsForOwnershipChange(HashSet<int> slots, string nodeid,
         }
 
         /// <summary>
-        /// Reset slot state to stable
+        /// Try to reset slot state to <see cref="SlotState.STABLE"/>
         /// </summary>
-        /// <param name="slot">Slot id to reset state</param>       
-        /// <param name="errorMessage"></param>
+        /// <param name="slot">Slot id to reset state</param>
         /// <returns>True on success, false otherwise</returns>
-        public bool TryResetSlotState(int slot, out ReadOnlySpan<byte> errorMessage)
```

Comment:
Should this method have error messages/failures 😅 Should we check `currentConfig.GetWorkerIdFromSlot` inside the loop?

Reply:
Check it for what? In general, callers to this method have to do validity checking.
This method tries to reset the slot state to stable if it was either in MIGRATING or IMPORTING state. The former means that the local node was the owner of the slot before it was set to MIGRATING, so we reset to 1.
In the latter situation, we still point to the old owner in the slotmap so we just need to get that info and set the node to STABLE.

PS: I have been trying to write cluster dev guide for a while now. Maybe this week, if PR reviews settle down 😄.
I am sure I am going to find couple of issues😅

Reply:
Alright, I don't have good mental model for all the cluster state transitions so I kinda hoped this could fail somehow so we could keep the `Try*` pattern and the nice switch expression 😄 ResetSlotState it is then?